Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM centos:5
```

Comment:
If this docker image is meant to run tests, we should use the latest stable CENTOS, in fact.